demmt/nvrm_decode_ioctl.c

```diff
@@ -173,6 +173,14 @@ static void decode_nvrm_ioctl_check_version_str(struct nvrm_ioctl_check_version_
 	mmt_log_cont("cmd: %d, reply: %d, vernum: %s\n", s->cmd, s->reply, s->vernum);
 }
 
+static void decode_nvrm_ioctl_xfer_cmd(struct nvrm_ioctl_xfer_cmd *s)
+{
+	nvrm_print_x32(s, cmd);
+	nvrm_print_x32(s, size);
+	nvrm_print_x64(s, ptr);
```

Comment:
Is the data behind "ptr" interesting? Is "size" the length of data behind "ptr"?
If the answer to both questions is "yes", then it's worth modifying mmt to dump it (see eg "handle_nvrm_ioctl_query") and print it here (see eg "decode_nvrm_query_object_classes").

Reply:
It's effectively another means of passing in the data for the ioctl handling code (you can use this to call any of the other ioctls indirectly).

Reply:
So it is useful. This can be addressed in the future though.

Reply:
Thanks!